pandas/core/indexes/multi.py

```diff
@@ -2721,8 +2723,7 @@ def _maybe_to_slice(loc):
             mask[loc] = True
             return mask
 
-        if not isinstance(key, (tuple, list)):
-            # not including list here breaks some indexing, xref #30892
```

Comment:
I think it is okay to remove this comment since we're not actually checking for list here, or should it be moved? cc @jbrockmendel

Reply:
i think youre right (though havent checked 30892)